File: langgraph/data/equiv/130.py
def generate_custom_sequence(length):
    if length == 0:
        return [1]
    
    sequence = [1, 3]
    index = 2
    
    while index <= length:
        if index % 2 == 0:
            sequence.append(index / 2 + 1)
        else:
            sequence.append(sequence[index - 1] + sequence[index - 2] + (index + 3) / 2)
        index += 1
    
    return sequence

# method name: tri -> generate_custom_sequence
# parameter name: n -> length
# variable name: my_tri -> sequence, i -> index
# changed for loop to while loop
# kept the same logic for appending values to the list

File: langgraph/result/qwen/14b-py2js/130.js
function tri(n) {
    if (n === 0) {
        return [1];
    }
    const myTri = [1, 3];
    for (let i = 2; i <= n; i++) {
        if (i % 2 === 0) {
            myTri.push(i / 2 + 1);
        } else {
            myTri.push(myTri[i - 1] + myTri[i - 2] + (i + 3) / 2);
        }
    }
    return myTri;
}